### ISSUE TITLE
HyperParameter Tuning

### ISSUE THREAD

**Issue by shivamprakashsp** [shivamprakashsp/Sentiment_Analysis#2]
The Bi-LSTM model used has not been tuned. 
The current val -accuracy is around 85%. Try using GridSearchCV with k =10,and tune hyperparameters like learning rate, optimizer, batch size and epochs.
Expected accuracy after tuning  - 88~90%
